pure2.md

```diff
@@ -0,0 +1,535 @@
+
+Each offline-update targets metadata file lists a set of images eligible for installation together as an offline update. This metadata is designed to be loaded onto a portable storage medium, such as a USB drive or a diagnostics/flashing tool with attached storage, alongside an up-to-date version of all non-device-specific metadata from both Uptane repositories and binaries for all targets listed. Uptane clients can check for the presence of offline-update metadata, either by an automated process (e.g. periodically checking some well-known location on a filesystem or TFTP server) or via a manually-triggered diagnostic command. If offline-update metadata is found, the clients validate it, check it against the Image repository metadata, and then install the targets.
+
+When a devices receives online updates from the Director receive the latest version of the offline-update snapshot metadata as part of the normal update cycle.
+
+### Offline-update role in root metadata
+
+To perform offline updates, the root metadata file in the Director repository SHALL include a new "offline-updates" role. This role MAY use the same signing key as the Targets role, but SHOULD use its own. The offline-update role's key(s) MAY be kept online or offline, depending on the security requirements of the implementor.
+
+### Offline-update snapshot metadata
+
+The offline-update snapshot metadata lists version numbers and filenames of all offline-update targets metadata files.
+
+For each offline-update targets metadata file on the repository, the offline-update snapshot metadata SHALL contain the following information:
+
+1.  The filename and version number of the offline-update targets metadata file.
+
+### Offline-update targets metadata
+
+Each offline-update targets metadata file on a repository contains information about a set of images eligible to be installed on devices. Like other Director targets metadata files, offline-update targets metadata files SHALL NOT contain delegations.
+
+When a device is validating offline-update targets metadata, it will only execute an update if it can unambiguously determine which images to install. For this reason, the metadata SHOULD include a mechanism to permit disambiguation in multi-ECU systems. See the deployment considerations section for more detail on this recommendation.
+
+### Addendum to Director repository snapshot metadata requirements
+
+Offline-update targets metadata files SHOULD NOT be listed in the main snapshots metadata file of the Director repository, if implemented. Because the offline-update role signs its own snapshot metadata, listing those files in the main snapshots metadata file is redundant, and increases the size of a file that is downloaded very frequently.
```

Comment:
I'm not a fan of this change, because it implies that "if implemented" is grammatically attached to "the Director repository". Although that reading is clearly nonsense to an experienced reader, I think my phrasing has less ambiguity.